lib/cheep.rb

```diff
@@ -0,0 +1,39 @@
+require 'pg'
+
+class Cheep
+
+  attr_reader :id, :author, :message
+
+  def initialize(id:, author:, message:)
+    @id = id
+    @author = author
+    @message = message
+
+  end
+
+  def self.all
+    if ENV['ENVIRONMENT'] == 'test'
+      connection = PG.connect(dbname: 'cheeps_test_database')
+    else
+      connection = PG.connect(dbname: 'cheeps')
+    end
+
+    result = connection.exec("SELECT * FROM cheeps;")
+    result.map do |cheep|
+      Cheep.new(id: cheep['id'], author: cheep['author'], message: cheep['message'])
```

Comment:
Since timestamp is already part of the db, you could map the timestamp to the model as well